sdk/schemaregistry/Azure.Data.SchemaRegistry/api/Azure.Data.SchemaRegistry.netstandard2.0.cs

```diff
@@ -5,8 +5,11 @@ namespace Azure.Data.SchemaRegistry
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
+        public SchemaFormat(Azure.Core.ContentType contentType) { throw null; }
```

Comment:
Hmm where is this coming from? I don't see it in the generated code.

Reply:
I forgot to re-export the API, it's gone now that I have